index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porofolio main page</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/091ffb7972.js" crossorigin="anonymous"></script>
+</head>
+<body>
+      <header class="head">
+        <nav>
+          <h2 class="pagename" >Welcome</h2>
+        </nav>
+        <img src="tools/Union.svg" alt="" class="nav-icon">
+    </header>
+    <section class="welcome">
+        <p ><span class="greetings">Hey there. I’m Josue</span><br>
+        <span class="profession">I’m a software developer</span></p>
+        <p class="description">I'm a student who is in his first week of the microverse module,
+             this is my first project I'm working on to see my advencements or progress.</p>
+           <footer class="socials">
+            <img src="tools/twitter.svg" alt="">
+            <img src="tools/Linkedin icon.svg" alt="">
+            <img src="tools/medium.svg" alt="">
+            <img src="tools/github.svg" alt="">
+            <img src="tools/angellist.svg" alt="">
+           </footer>
```

Comment:
- [ ] Kindly use `ul` and `li` tags instead of `footer` tag as you can see below due to SEO purpose 👍 and wrap all your images with `a` tags because they are links to your social media accounts 🤔

```
<ul class="socials">
  <li><a href="#"><img src="tools/twitter.svg" alt=""></a></li>
  <li><a href="#"><img src="tools/Linkedin icon.svg" alt=""></a></li>
  <li><a href="#"><img src="tools/medium.svg" alt=""></a></li>
  <li><a href="#"><img src="tools/github.svg" alt=""></a></li>
  <li><a href="#"><img src="tools/angellist.svg" alt=""></a></li>
</ul>
```